Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -1,3 +1,5 @@
+#: usage: brew gist-logs [--new-issue|-n] <formula>
```

Comment:
Can you expand on this and make it more like those special documentation comments for other commands, e.g. [`deps`](https://github.com/Homebrew/brew/blob/504152038cb591a661721b09e989f428bbc6ae0a/Library/Homebrew/cmd/deps.rb#L1-L36), i.e. explain the purpose and document the options/arguments? What you see there is basically a snippet of Markdown as understood by `ronn`. To get the correct output, both the indentation and the formatting need to be matched exactly.

You'll need to check that the output of `brew help deps` and `brew help gist-logs` looks similar in terms of formatting. As a further step, you'll need to run `brew man` to regenerate the man page and then check the formatting of `man brew`.